src/nvim/api/vim.c

```diff
@@ -1562,3 +1563,21 @@ Object nvim_get_proc(Integer pid, Error *err)
 #endif
   return rvobj;
 }
+
+/// Give the info of the state of the Caps/Num/Scroll lock.
+///
+/// This info will be in form of the dictionary, having
+/// `CapsLock`, `NumLock` and `ScrollLock` as the entities
+/// iff they are supported by their respective OS and are implemented.
```

Comment:
Respective to what? And it is not OS-specific in longer term, it is implementation-specific. Just say about checking for them being implemented.

Reply:
Yes, now, it was a previous comment